test/lib/audit_log/middleware_test.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'audit_log/audit_log'
```

Comment:
I suppose this is the `require` intended to be used within `application.rb`?

Reply:
Yes.